torchao/quantization/utils.py

```diff
@@ -456,3 +457,17 @@ def per_token_dynamic_quant(input: torch.Tensor) -> torch.Tensor:
         input, scales, zero_points, quant_min, quant_max, torch.int8, orig_dtype
     )
     return input.to(orig_dtype)
+
+def recommended_inductor_config_setter():
+    """
+    Set inductor config to use the following optimizations:
+        coordinate_descent_tuning = True
+        force_fuse_int_mm_with_mul = True
+        fx_graph_cache = True  
+        triton.unique_kernel_names = True
+    """
+    torch._inductor.config.coordinate_descent_tuning = True
+    torch._inductor.config.force_fuse_int_mm_with_mul = True  
```

Comment:
So for BC there should be 2 flags we support now https://github.com/pytorch/ao/commit/664f07393f761cb681466f769ab74de291ffc6a8

Reply:
that's a different flag